torchinductor/codegen/common.py

```diff
@@ -20,7 +20,7 @@
 log = logging.getLogger(__name__)
 
 
-def _simplify_loops(index_vars, sizes, index_formulas):
+def _simplify_loops(index_vars, sizes, index_formulas, preserver_reoredering=False):
```

Comment:
I remove this option and use Jason's suggestion https://github.com/pytorch/torchdynamo/blob/daa476aeb35d9cadd44f30b3c557584323fc25a0/torchinductor/codegen/triton.py#L580-L581 to prevent reordering